avm/res/resources/deployment-script/tests/e2e/defaults/main.test.bicep

```diff
@@ -4,6 +4,7 @@ metadata name = 'Using only defaults'
 metadata description = '''
 This instance deploys the module with the minimum set of required parameters.
 > **Note:** The test currently implements additional non-required parameters to cater for a test-specific limitation.
```

Comment:
```suggestion
```

This can acutally be removed now. Please note that you'd need to refresh the readme